docs/src/components/productDesignKit/DesignKitValues.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import GradientText from 'docs/src/components/typography/GradientText';
+import SvgPalette from 'docs/src/icons/SvgPalette';
+import SvgDocs from 'docs/src/icons/SvgDocs';
+import SvgCode from 'docs/src/icons/SvgCode';
+
+const content = [
+  {
+    icon: <SvgPalette />,
+    title: 'For designers',
+    description:
+      "Save time getting the Core components all set up in your favorite design tool. Customize them however you want to match to your product's branding.",
+  },
+  {
+    icon: <SvgDocs />,
+    title: 'For product managers',
+    description:
+      'Quickly put together ideas and mockups with already in hand components from your actual product.',
+  },
+  {
+    icon: <SvgCode />,
+    title: 'For developers',
+    description:
+      'Effortlessly communicate with designers using the same language around the Core components.',
+  },
+];
+
+const DesignKitValues = () => {
+  return (
+    <Container sx={{ py: { xs: 4, sm: 6, md: 8 } }}>
+      <Typography variant="body2" color="primary" fontWeight="bold">
+        Why go with us?
```

Comment:
Shouldn't this be the heading and the text below the introductory text for the section?

Reply:
Are you concern about a11y or just the design?

Reply:
Both.

Reply:
This is a pattern used in a lot of websites out there: a section caption (or id, if you will), a heading, and a description. Design/product-wise, the caption section is meant to be a call out for the heading, almost like a question that the heading will answer (Why go with us? -> ... / What do you get? -> ...). Not all of them follow strictly this but you get the idea.

On the code side of things, it's actually interesting to see two different players using this same pattern in different ways. 
<img width="500" alt="Screen Shot 2021-08-18 at 15 52 28" src="https://user-images.githubusercontent.com/67129314/129955513-170617dc-55e2-4189-8525-fb0ffec59fe2.png">

What I mean by different is that they use different tags for the same design basically. If I'm not mistaken, the `body2` variant maps by default to a span tag. We could have all texts that represent _a section caption_ use another tag using the `component` prop or change to which tag the body2 variant maps in the theme. I'm not sure which of these is the best way forward and neither what tag is best suitable for using though 😅

Reply:
me either.

Reply:
> This is a pattern used in a lot of websites out there:

And this tells us what?

Either way, "Why go with us" is clearly the heading and "Be more efficient designing and developing with the same library." the introductory sentence. The markup needs to reflect that.

Reply:
> This is a pattern used in a lot of websites out there:

And this tells us what?

Either way, "Why go with us" is clearly the heading and "Be more efficient designing and developing with the same library." the introductory sentence. The markup needs to reflect that.